src/pkgcheck/checks/network.py

```diff
@@ -208,7 +208,7 @@ def schedule(self, pkg, executor, futures):
             if url.startswith('ftp://'):
                 self._schedule_check(
                     self._ftp_check, attr, url, executor, futures, pkg=pkg)
-            else:
+            elif url.startswith('https://') or url.startswith('http://'):
```

Comment:
str.startswith() can take a tuple of prefixes to check for so I'd simplify it to one call.